src/Tools/dotnet-monitor/LoggingEventIds.cs

```diff
@@ -106,7 +106,9 @@ internal enum LoggingEventIds
         ExtensionManifestNotParsable = 94,
         ExtensionMalformedOutput = 95,
         EgressProviderTypeNotExist = 96,
-        ProfilerRuntimeIdentifier = 97
+        ProfilerRuntimeIdentifier = 97,
+        EndpointInitializationFailed = 98,
```

Comment:
Can you start at 99? 98 is taken in https://github.com/dotnet/dotnet-monitor/pull/4792/files, but hasn't been merged to main yet.